src/pages/PageNotFound404.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default function PageNotFound404() {
+  /************************************
+   * Render
+   ************************************/
+
+  return (
+    <>
```

Comment:
You forgot to remove the React.fragment tag ```<>```

Reply:
Ah, so that's what that is